docs/design/latency_definitions/callback.md

```diff
@@ -0,0 +1,40 @@
+# Callback
+
+The callback latency is the time difference from callback start to callback end.
```

Comment:
```suggestion
Callback latency is defined as duration between beginning and end of callback execution.
```

Reply:
`Time difference` has meaning for expressing difference between two time zone. It is not appropriate word here.
https://www.collinsdictionary.com/jp/dictionary/english/time-difference

Reply:
addressed: [ea6f556](https://github.com/tier4/CARET_doc/pull/106/commits/ea6f556a84802265fdd6222bee05a63aa0620a68)